specification/semantic_conventions.md

```diff
@@ -27,3 +28,41 @@ All Splunk distributions of OpenTelemetry,
+
+## Real User Monitoring Spans and Attributes
+
+Real User Monitoring (RUM) libraries MUST set the `service.name` resource attribute to the value of the `applicationName`
+configuration property. This is the only resource attribute that RUM libraries are supposed to set because it's the only one
+the Zipkin exporter can understand.
+
+The following attributes MUST be added to all spans produced by RUM libraries:
+
+| Name                  | Type   | Description                                                                                                       |
+| ----                  | ----   | -----------                                                                                                       |
+| `app`                 | string | The value of the `applicationName` property configured in `SplunkRum`; same as `service.name` resource attribute. |
+| `splunk.rumSessionId` | string | The RUM session ID                                                                                                |
+| `splunk.rum.version`  | string | Version of the RUM library                                                                                        |
+| `component`           | string | Name of the instrumentation that produced this span                                                               |
+
+The following attributes MAY be added to all spans emitted by RUM libraries:
+
+| Name                     | Type   | Description                                                                     |
+| ----                     | ----   | -----------                                                                     |
+| `deployment.environment` | string | Value of the `deploymentEnvironment` property configured in `SplunkRum`, if any |
+
+Mobile RUM libraries (iOS, Android) MUST add device and system information to all spans:
+
+| Name                | Type   | Description        |
+| ----                | ----   | -----------        |
+| `device.model.name` | string | Name of the device |
+| `os.type`           | string | `iOS` or `Android` |
+| `os.version`        | string | Version of the OS  |
+
+Web RUM library MUST add the following attributes to all spans:
+
+| Name                     | Type   | Description                                                                                                                    |
+| ----                     | ----   | -----------                                                                                                                    |
+| `location.href`          | string | Value of [`location.href`](https://developer.mozilla.org/en-US/docs/Web/API/Location/href) at the moment of creating the span. |
+| `splunk.script_instance` | string | A 64bit identifier, unique to every instance of the `SplunkRum` script.                                                        |
```

Comment:
Technically it's:
> every instance of `SplunkRum` library.
Because if you add the script twice, the library is still going to run once.